python_backend/api/v1/endpoints/kb.py

```
@@ -0,0 +1,87 @@
+class UploadResponse(BaseModel):
+    tenant_id: str
+    doc_id: str
+    chunks: int
+
+
+def _extract_text(filename: str, data: bytes) -> str:
+    name = (filename or "").lower()
+    if name.endswith(".txt"):
+        try:
+            return data.decode("utf-8", errors="ignore")
+        except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)